<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CollectionDialog</class>
 <widget class="QDialog" name="CollectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Add Collection</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>14</number>
     </property>
     <property name="verticalSpacing">
      <number>14</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Collection Title</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="collectionTitle"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Year</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="collectionYearSpinbox">
       <property name="minimum">
        <number>1900</number>
       </property>
       <property name="maximum">
        <number>2100</number>
       </property>
       <property name="value">
        <number>2020</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Quantity</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="collectionQuantitySpinbox">
       <property name="maximum">
        <number>10000</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QTextEdit" name="collectionDescriptionTextEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Genres</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QListWidget" name="collectionGenreList">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>Action</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Adventure</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Alternative Rock</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Anime</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Biography</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Christian Gospel</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Classic</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Comedy</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Comic Book</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Country</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Drama</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>EDM</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fantasy</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Graphic Novel</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Historical</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Horror</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Indie Rock</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Jazz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Latin</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Literary Fiction</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mystery</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Poetry</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Rap</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Rhythm and Blues</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Rock</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Romance</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Science Fiction</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Short Story</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Suspense</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Techno</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Western</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>14</number>
     </property>
     <property name="bottomMargin">
      <number>14</number>
     </property>
     <item>
      <widget class="QPushButton" name="addBookPushButton">
       <property name="text">
        <string>Add Book</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addCdPushButton">
       <property name="text">
        <string>Add Cd</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addDvdPushButton">
       <property name="text">
        <string>Add Dvd</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="acceptButtonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>acceptButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>CollectionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>402</x>
     <y>570</y>
    </hint>
    <hint type="destinationlabel">
     <x>500</x>
     <y>442</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>acceptButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>CollectionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>402</x>
     <y>570</y>
    </hint>
    <hint type="destinationlabel">
     <x>500</x>
     <y>442</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
